.github/workflows/legacy-continuous-integration.yml

```
@@ -0,0 +1,47 @@
+        php-version:
+          - '7.0'
+          - '7.1'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup PHP (legacy)
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-version }}
+          composer: '2.2'  # 古いPHPで動くComposerを固定
```

actionlint (1.7.4)

input "composer" is not defined in action "shivammathur/setup-php@v2". available inputs are "coverage", "extensions", "ini-file", "ini-values", "php-version", "php-version-file", "tools"

(action)